bin/git-fork

```diff
@@ -14,10 +14,12 @@ test -z "$url" && abort "github repo needs to be specified as an argument"
 echo "Enter your github username"
 read user
 [ -n "$user" ] || abort "git username required"
-echo "Enter github password for user \"$user\""
-read -s passwd
-echo "Enter github two-factor authentication code (leave blank if not set up)"
-read MFA_CODE
+# personal access token
+# config name is github-personal-access-token '_' is not allowed in git config
+
+github_personal_access_token=$(git config git-extras.github-personal-access-token)
```

Comment:
Please update the doc like this:
https://github.com/tj/git-extras/blob/master/man/git-pull-request.md#description

Reply:
Okay...updated.